recipe/activate.bat

```diff
@@ -156,6 +171,13 @@ if %ERRORLEVEL% neq 0 (
 )
 popd
 
+:: unset auxiliary variables
+set "CMAKE_GEN="
```

Comment:
It looks like CMAKE_GEN is being by used conda-forge recipes even though it's not supposed to be.

Reply:
It looks like that's [limited](https://github.com/search?q=org%3Aconda-forge+%2FCMAKE_GEN%5Cb%2F&type=code) to 3 feedstocks. Sounds like we could just fix those?

Reply:
Yeah, let's fix those first. I'd hate to break `scikit-build-core` package.since it's used in an activate script.